aiohttp/connector.py

```diff
@@ -781,18 +791,13 @@ def _get_ssl_context(self, req):
         if req.is_ssl():
             if ssl is None:  # pragma: no cover
                 raise RuntimeError('SSL is not supported.')
-            sslcontext = req.ssl
-            if isinstance(sslcontext, ssl.SSLContext):
-                return sslcontext
-            if sslcontext is not None:
-                # not verified or fingerprinted
-                return self._make_ssl_context(False)
-            sslcontext = self._ssl
-            if isinstance(sslcontext, ssl.SSLContext):
-                return sslcontext
-            if sslcontext is not None:
-                # not verified or fingerprinted
-                return self._make_ssl_context(False)
+
+            if req.ssl is not None:
```

Comment:
smth like this would look nicer that duplicate if-clauses:
```python
_ssl = req.ssl or self._ssl
if _ssl is not None:
    return self._check_ssl(_ssl)
```